src/config/config.js

```diff
@@ -43,11 +43,16 @@ let _blockchains = {
         chainId: '39f5e2ede1f8bc1a3a54a7914414e3779e33193f1f5693510e73cb7a87617447',
         nodeList: [
             {
-                url: "wss://testnet.nodes.bitshares.ws",
```

Comment:
Could you please add https://eu.nodes.testnet.bitshares.ws/

Reply:
https or wss?